app/src/main/java/com/cornellappdev/transit/ui/components/MenuItem.kt

```diff
@@ -24,16 +28,17 @@ import androidx.compose.ui.unit.sp
  * @param sublabel The sublabel for the item
  */
 @Composable
-fun MenuItem(icon: ImageVector, label: String, sublabel: String) {
+fun MenuItem(icon: ImageVector, label: String, sublabel: String, onClick: () -> Unit) {
     Row(Modifier.fillMaxWidth(), verticalAlignment = Alignment.CenterVertically) {
         Icon(
             icon,
             contentDescription = "Place",
-            modifier = Modifier.padding(horizontal = 10.dp)
+            modifier = Modifier.padding(horizontal = 10.dp),
+            tint = transitBlue
         )
```

Comment:
I see a slight design inconsistency here. The padding on the right of the icon should be 20, not 10 (but the left is probably still supposed to be 10).
![image](https://github.com/cuappdev/ithaca-transit-compose/assets/47724806/13142243-fa5f-4fac-972f-a380e8fd3cfa)
Above, you can see that there's definitely more space in the design than here. In Figma, you can enable dev mode and click around in the designs to see these padding values for yourself.

Reply:
Fixed! here is updated screenshot :) I also changed the font sizes after looking @ dev mode 
![image](https://github.com/cuappdev/ithaca-transit-compose/assets/63558657/4c18ba2a-b5f7-4182-ac83-8a076a3c9ae8)